### ISSUE TITLE
Not working on Window 10 17134, no error message, just nothing happening

### ISSUE THREAD

**Issue by marvinb89** [shadowsocks/shadowsocks-windows#2090]
### Shadowsocks version / 影梭版本
4.1.2 / then tried 4.1.1

### Environment(Operating system, .NET Framework, etc) / 使用环境（操作系统，.NET Framework等）
Windows 10 (Build 17134)

### Steps you have tried / 操作步骤
Delete all files, run as administrator, double double check all the settings, netsh interface ipv4 reset
netsh interface ipv6 reset, netsh winsock reset, restart windows. 

### What did you expect to see? / 期望的结果
I expected to see the IP of my VPS when I go on ifconfig.co

### What did you see instead? / 实际结果
My chinese IP. On my phone, with the android app, it works fine, no problems there. 

### Config and error log in detail (with all sensitive info masked) / 配置文件和日志文件（请隐去敏感信息）

  "configs": [
    {
      "server": "XX",
      "server_port": XX,
      "password": "XX",
      "method": "aes-256-cfb",
      "plugin": "",
      "plugin_opts": "",
      "plugin_args": "",
      "remarks": "",
      "timeout": 5
    }
  ],
  "strategy": null,
  "index": 0,
  "global": false,
  "enabled": false,
  "shareOverLan": false,
  "isDefault": false,
  "localPort": 1080,
  "portableMode": true,
  "pacUrl": null,
  "useOnlinePac": false,
  "secureLocalPac": true,
  "availabilityStatistics": false,
  "autoCheckUpdate": true,
  "checkPreRelease": false,
  "isVerboseLogging": true,
  "logViewer": {
    "topMost": false,
    "wrapText": false,
    "toolbarShown": false,
    "Font": "Consolas, 8pt",
    "BackgroundColor": "Black",
    "TextColor": "White"
  },
  "proxy": {
    "useProxy": false,
    "proxyType": 0,
    "proxyServer": "",
    "proxyPort": 0,
    "proxyTimeout": 3
  },
  "hotkey": {
    "SwitchSystemProxy": "",
    "SwitchSystemProxyMode": "",
    "SwitchAllowLan": "",
    "ShowLogs": "",
    "ServerMoveUp": "",
    "ServerMoveDown": "",
    "RegHotkeysAtStartup": false
  }
}

The log file does not have more than this: 

[2018-11-28 22:56:33] Shadowsocks started (4.1.1)
[2018-11-28 22:56:33] 
=========================
Registered Encryptor Info
aes-128-cfb=>StreamOpenSSLEncryptor
aes-192-cfb=>StreamOpenSSLEncryptor
aes-256-cfb=>StreamOpenSSLEncryptor
aes-128-ctr=>StreamOpenSSLEncryptor
aes-192-ctr=>StreamOpenSSLEncryptor
aes-256-ctr=>StreamOpenSSLEncryptor
bf-cfb=>StreamOpenSSLEncryptor
camellia-128-cfb=>StreamOpenSSLEncryptor
camellia-192-cfb=>StreamOpenSSLEncryptor
camellia-256-cfb=>StreamOpenSSLEncryptor
rc4-md5=>StreamOpenSSLEncryptor
salsa20=>StreamSodiumEncryptor
chacha20=>StreamSodiumEncryptor
chacha20-ietf=>StreamSodiumEncryptor
aes-128-gcm=>AEADOpenSSLEncryptor
aes-192-gcm=>AEADOpenSSLEncryptor
aes-256-gcm=>AEADOpenSSLEncryptor
chacha20-ietf-poly1305=>AEADOpenSSLEncryptor
xchacha20-ietf-poly1305=>AEADSodiumEncryptor
=========================

**Comment by chenshaoju** [comment]
When started shadowsocks.exe, right click Shadowsocks in your taskbar tray icon area, And make sure has selected `Enable System Proxy`.
If still not working, Please right click Shadowsocks in your taskbar tray icon area again, Moving to PAC, And click `Update Local PAC from GFWList`, Waiting for a `PAC updated` bubble message pop-up.
Then should work.

**Comment by marvinb89** [comment]
Thanks it works now. BUT tbh I think it was rather my own fault. I did not know the differences between PAC and global in the mode options so I was on PAC and thought it doesn't work because I used certain websites to check my IP address and it still showed my Chinese IP address and that made me believe it doesn't work. Now with global mode, it works! So not sure if it was your answer that helped but THANKS for the reply and support, appreciate it! 谢谢你！
